scripts/merge-gisaid-owid.py

```diff
@@ -225,14 +244,15 @@ def main(args_list=None):
     merged_df = merge_gisaid_owid(gisaid_country_variants_df, owid_df)
     print('Pivoting merged data...')
     merged_pivoted_df = pivot_merged_df(merged_df)
+    print('Add WHO regions to countries...')
+    merged_pivoted_df = add_regions(merged_pivoted_df)
     print('Done.')
 
-    # only output dates up until the latest GISAID available (collection dates will lag a few days)
-    max_gisaid_date = gisaid_country_variants_df.collect_date.max()
-    merged_pivoted_df_latest = merged_pivoted_df.loc[merged_pivoted_df.owid_date <= max_gisaid_date]
+    max_gisaid_date = gisaid_df.submit_date.max()
+    merged_pivoted_df_latest = merged_pivoted_df.loc[(merged_pivoted_df.owid_date <= max_gisaid_date)
+                                                      &(merged_pivoted_df.owid_date>='2020-01-01')]
```

Comment:
since we're filtering to 2019-12-01 elsewhere, should we use that same filter here?

Reply:
yup, agreed, made that change